docs/configuration/radio/device.mdx

```diff
@@ -80,7 +80,7 @@ This is the GPIO pin number that will be used for the PWM buzzer, if your device
 
 ### Node Info Broadcast Seconds
 
-This is the number of seconds between NodeInfo message (containing i.a. long and short name) broadcasts from the device. The device will still respond ad-hoc to NodeInfo messages when a response is wanted. When the device hears any packet from a node it doesn't know yet, it will send its NodeInfo and ask for a response automatically.
+This is the number of seconds between NodeInfo message (containing i.a. long and short name) broadcasts from the device. The device will still respond ad-hoc to NodeInfo messages when a response is wanted. When the device hears any packet from a node it doesn't know yet, it will send its NodeInfo and ask for a response automatically. Minimum is one hour and will never broadcast more than once every five minutes.
```

Comment:
"Minimum is ..." sounds like it will sent this at least every hour. I would suggest to put "The minimum you can set this to is one hour." after the first sentence, and then adding "... if it didn't do so already in the last five minutes." to the last sentence.

Reply:
@nagumii just tagging you in case you missed this.